packages/p2p/src/components/buy-sell/popup.jsx

```diff
@@ -24,15 +24,15 @@ class Popup extends Component {
 
         const order = await requestWS({
             p2p_order_create: 1,
+            advert_id: ad.id,
             amount: values.amount,
-            offer_id: ad.offer_id,
         });
 
         if (!order.error) {
-            const order_info = await requestWS({ p2p_order_info: 1, order_id: order.p2p_order_create.order_id });
+            const order_info = await requestWS({ p2p_order_info: 1, id: order.p2p_order_create.id });
+            this.props.handleClose();
```

Comment:
Why was `this.props.handleClose()` moved up?

Reply:
because it updates the state, and we can't do that after redirecting away. it gives a console error.